patches/third_party/WebKit/filter.patch

```diff
@@ -0,0 +1,2443 @@
+diff --git a/base/BUILD.gn b/base/BUILD.gn
```

Comment:
this file doesn't look correct to me. It seems to have changes from things outside of webkit